### ISSUE TITLE
ctrl-enter or command-enter should not add a line

### ISSUE THREAD

**Issue by davidbau** [droplet-editor/droplet#103]
When ctrl- (non-mac) or command- (on mac) is held down when keying, it should not type letters, because they are used as command keys by applications.

In particular, command-enter should not insert extra blank line blocks in block mode.  Pencil Code uses command-enter to run a program.

**Comment by xinan** [comment]
Do we use any keyboard shortcut libraries?

**Comment by davidbau** [comment]
No, in pencil code we don't.  I think we don't in droplet either; anthony can verify.

**Comment by sakagg** [comment]
While we are at it, how about adding newlines with shift+enter.

**Comment by davidbau** [comment]
+1 on shift-enter.  Important for HTML mode.
